app/activity/schema.ts

```diff
@@ -0,0 +1,34 @@
+import { zodResolver } from '@hookform/resolvers/zod'
+import * as z from 'zod'
+
+const activitySchema = z
+  .object({
+    title: z.string().min(1, { message: 'Please enter a title' }),
+    timeFrom: z.date({
+      required_error: 'Please select a date from',
+      invalid_type_error: "That's not a date"
+    }),
+    timeTo: z.date().nullable(),
+    address: z.string().min(0).max(50).nullable(),
+    url: z.union([z.string().url().nullish(), z.literal('')]),
+    memo: z.string().min(0).max(300).nullable(),
+    cost: z.string().nullable(),
```

Comment:
When I left empty input value, I got an server error! I think the type does not match?
<img width="796" alt="Screenshot 2024-02-26 at 12 32 24 PM" src="https://github.com/tabi-memo/frontend/assets/66394413/38831a65-76c9-4214-9f3f-66b6ed008096">

Reply:
This problem is the same error that occurs when Trip create/edit.
![Screenshot 2024-03-01 at 13 21 54](https://github.com/tabi-memo/frontend/assets/7115171/05daff32-a799-4294-b38e-28507ecd02ec)


And the error message `"invalid input syntax for type numeric: """` occurs when an attempt is made to insert an empty string ("") into a numeric field in a database such as PostgreSQL.
So, that's why I fixed trip and activity custom hook functions.
https://github.com/tabi-memo/frontend/pull/34/commits/0829cc9423b863ff6eaff7e3c7e82f2f83257ee7